install/kubernetes/cilium/values.yaml

```diff
@@ -515,3 +509,50 @@ global:
 
   cnpStatusUpdates:
     enabled: false
+
+# Configures the hubble-cli subchart
+hubble-cli:
```

Comment:
i have a question about this, what's the difference between this section and cilium/charts/hubble-cli/values.yaml? if i'm adding a new configuration parameter, do i need to add it to both places?

Reply:
makes sense thanks for the explanation!